File: Java/Introduction/WhatIsYourNameAndAge.java
import java.util.Scanner;
public class WhatIsYourNameAndAge{
	 public static void main(String[] args) {
		 Scanner myObj = new Scanner(System.in);
		 System.out.println("Enter your Name");
		 String name = myObj.nextLine();
		 System.out.println("Enter your Age");
		 String age = myObj.nextLine();
		 System.out.println("Hello " + name + ". Your age is " + age);
	 }
}
		 

File: C++/Introduction/WhatIsYourNameAndAge.cpp
#include <iostream>
#include <string>

using namespace std;
int main() {
	string nameper;
	int ageper;
	cout << "Enter your name";
	getline(cin, nameper);
	cout << "Enter your age";
	cin >> ageper;
	cout << "Hello " << nameper << ". Your age is " << ageper;
}
